src/trg-main-window.c

```diff
@@ -2570,27 +2452,11 @@ void trg_main_window_add_status_icon(TrgMainWindow * win)
         app_indicator_set_status(priv->appIndicator,
```

Comment:
I recommend a guard here to ensure `priv->appIndicator` is never set twice.

(It wasn't in my testing so that isn't why it won't disappear)